convert2rhel/subscription.py

```diff
@@ -207,6 +252,24 @@ def register_system():
     return None
 
 
+def refresh_subscription_info():
+    """
+    Have subscription-manager pull new subscription data from the server.
+
+    A byproduct of refreshing is that subscription-manager will reexamine the filesystem for the
+    RHSM product certificate.  This is the reason that we need to call this function.
+    """
+    cmd = ["subscription-manager", "refresh"]
+    output, ret_code = utils.run_subprocess(cmd, print_output=True)
```

Comment:
Do we really want to show the output after executing the command or is it only by debug purposes?

Reply:
You are right.  This is fixed.